modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets.adoc_{context}"]
+= Create a Secret object by using the web console
+
+As an administrator, you can create secrets using the web console. 
+
+. Navigate to *Workloads* → *Secrets*.
+. Click *Create* → *From YAML*.
+.. Edit the YAML manually to your specifications, or drag and drop a file into the YAML editor. 
+For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+ name: example
+ namespace: default
+type: Opaque <1>
+stringData:
+  username: <username>
+  password: <password>
+----
+<1> Specifies an opaque secret.
+
+. Click *Create*.
+. Click *Add Secret to workload*.
+.. From the drop-down menu, select the workload to add.
+.. Click *Save*.
+
+The following sections describe how to create an opaque secret, a service account token secret, a basic authentication secret, an SSH authentication secret, and a Docker configuration secret.  
```

Comment:
```suggestion
The following sections describe how to create an opaque secret, a service account token secret, a basic authentication secret, an SSH authentication secret, and a Docker configuration secret using CLI. You can alternatively use the web console to create these secrets.   
```

Reply:
One small suggestion :)

Reply:
Less certain about the "Alternatively use the web console to create these secrets" bit.

Reply:
Thanks Sara! I added the "CLI" bit and held off on the "Alternatively use the web console to create these secrets" bit.